src/zenml/cli/__init__.py

```diff
@@ -1596,7 +1596,6 @@ def my_pipeline(...):
 from zenml.cli.base import *  # noqa
 from zenml.cli.code_repository import *  # noqa
 from zenml.cli.config import *  # noqa
-from zenml.cli.example import *  # noqa
```

Comment:
We should remove the `zenml example` documentation from the huge docstring at the top of this file too.

Reply:
Ah, great catch, almost forgot that I removed `zenml example ...` as part of this PR 😅

Reply:
I'll remove all references 👍